packages/eui/src/components/form/switch/switch.tsx

```diff
@@ -47,7 +47,7 @@ export type EuiSwitchProps = CommonProps &
     compressed?: boolean;
     type?: 'submit' | 'reset' | 'button';
     /**
-     * Object of props passed to the label's <span/>
+     * Object of props passed to the label's `<span/>`
```

Comment:
minor syntax nit while we're here!
```suggestion
     * Object of props passed to the label's `<span />`
```